unit_test/solver/network/assets/test_solver_pipe.py

```diff
@@ -404,7 +404,7 @@ def test_calculate_heat_transfer_coefficient_fluid_large_reynolds(self) -> None:
         )
 
         # assert
-        self.assertAlmostEqual(heat_transfer_coefficient, 244.77, 2)
+        self.assertAlmostEqual(heat_transfer_coefficient, 246.9676649954736, 2)
```

Comment:
Why give so many decimals for this almost equal comparison when you only compare two digits? See other comparisons as well.

Reply:
adjusted